features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -874,6 +881,10 @@
 
     <SelfRegistration>
         <Enable>{{identity_mgt.user_self_registration.allow_self_registration}}</Enable>
+        <RegisterWithVerifiedChannels>{{identity_mgt.user_self_registration.register_with_verified_channels}}</RegisterWithVerifiedChannels>
```

Comment:
We may need to come up with a better name for this configuration

Reply:
Fixed : d47fd742281a832a7570cade4cd44c63c87483b0